src/util-inl.h

```diff
@@ -180,6 +180,11 @@ inline v8::Local<v8::String> OneByteString(v8::Isolate* isolate,
       .ToLocalChecked();
 }
 
+inline v8::Local<v8::String> OneByteString(v8::Isolate* isolate,
+                                           const std::string_view& str) {
```

Comment:
No need for const and reference, by default std string view doesn't copy

```suggestion
                                           std::string_view str) {
```

Reply:
But there's really no harm with the const ref here. I generally prefer this pattern.

Reply:
it sends a wrong message to a new C++ developer, hinting them towards std::string_view holding the underlying data structure.